collectors/logsmanagement.plugin/plugin_logsmanagement_generic.h

```diff
@@ -0,0 +1,37 @@
+/** @file plugins_logsmanagement_generic.h
+ *  @brief Incudes the structure and function definitions to use generic log charts.
+ *
+ *  @author Dimitris Pantazis
+ */
+
+#ifndef PLUGIN_LOGSMANAGEMENT_GENERIC_H_
+#define PLUGIN_LOGSMANAGEMENT_GENERIC_H_
+
+#include "../../daemon/common.h"
+#include "../../logsmanagement/file_info.h"
+#include "../../logsmanagement/circular_buffer.h"
+
+typedef struct Chart_data_generic chart_data_generic_t;
+
+#include "plugin_logsmanagement.h"
+
```

Comment:
please, remove next comments and if necessary add in external file as I already commented.

Reply:
Done!